internal/services/compute/orchestrated_virtual_machine_scale_set_resource.go

```diff
@@ -351,6 +380,14 @@ func resourceOrchestratedVirtualMachineScaleSetCreate(d *pluginsdk.ResourceData,
 		props.Sku = sku
 	}
 
+	if v, ok := d.GetOk("sku_profile"); ok {
+		if props.Sku == nil || pointer.From(props.Sku.Name) != "Mix" {
```

Comment:
Also, can we have this (`"Mix"`) defined as a `const` rather than using magic string everywhere?
maybe:
```suggestion
		if props.Sku == nil || pointer.From(props.Sku.Name) != skuNameMix {
```

Reply:
defined it in a const variable. Thanks.